tests/testsuite/namespace.go

```diff
@@ -200,12 +201,23 @@ func removeNamespaces() {
 
 	// Wait until the namespaces are terminated
 	fmt.Println("")
-	for _, namespace := range TestNamespaces {
-		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
+	for _, namespaceName := range TestNamespaces {
+		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespaceName)
 		EventuallyWithOffset(1, func() error {
-			return virtCli.CoreV1().Namespaces().Delete(context.Background(), namespace, metav1.DeleteOptions{})
-		}, 240*time.Second, 1*time.Second).Should(SatisfyAll(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())), fmt.Sprintf("should successfully delete namespace '%s'", namespace))
+			return virtCli.CoreV1().Namespaces().Delete(context.Background(), namespaceName, metav1.DeleteOptions{})
+		}, 240*time.Second, 1*time.Second).Should(SatisfyAll(HaveOccurred(), WithTransform(errors.IsNotFound, BeTrue())), fmt.Sprintf("should successfully delete namespace '%s': \n%s", namespaceName, dumpNamespaceObject(namespaceName)))
```

Comment:
I think `%s': \n%s", namespaceName, dumpNamespaceObject(namespaceName)` can be part of what dumpNamespaceObject prints, so you will need to just call it here
(i.e print there namespaceName)
btw maybe since you print the whole yaml you don't need to print namespaceName in additon at all

Reply:
good point. Done